src/pages/ListPage/ListPage.tsx

```diff
@@ -194,48 +212,59 @@ const ListPage: React.FunctionComponent<unknown> = () => {
             <Main>
-                { getPoliciesQuery.hasPolicies === false ? (
-                    <ListPageEmptyState
-                        createPolicy={ canWritePolicies ? toolbarActions.createCustomPolicy : undefined }
+                { !hasSystems ? (
+                    <AsynComponent
+                        appName="dashboard"
+                        module="./AppZeroState"
+                        scope="dashboard"
+                        ErrorComponent={ <ErrorState /> }
+                        app="Policies"
                     />
-                ) : (
-                    <Section>
-                        <PolicyToolbar
-                            ouiaId="main-toolbar"
-                            onCreatePolicy={ canWritePolicies ? toolbarActions.createCustomPolicy : undefined }
-                            onDeletePolicy={ canWritePolicies ? toolbarActions.onDeletePolicies : undefined }
-                            onEnablePolicy={ canWritePolicies ? toolbarActions.onEnablePolicies : undefined }
-                            onDisablePolicy={ canWritePolicies ? toolbarActions.onDisablePolicies : undefined }
-                            onPaginationChanged={ policyPage.changePage }
-                            onPaginationSizeChanged={ policyPage.changeItemsPerPage }
-                            onSelectionChanged={ policyRows.onSelectionChanged }
-                            selectedCount={ policyRows.selectionCount }
-                            page={ policyPage.page.index }
-                            pageCount={ policyRows.rows.length }
-                            perPage={ policyPage.page.size }
-                            showPerPageOptions={ true }
-                            filters={ policyFilters.filters }
-                            setFilters= { policyFilters.setFilters }
-                            clearFilters={ policyFilters.clearFilter }
-                            count={ getPoliciesQueryCount }
-                            onExport={ toolbarActions.onExport }
-                            showBottomPagination={ true }
-                        >
-                            <PolicyTable
-                                ouiaId="main-table"
-                                policies={ policyRows.rows }
-                                onCollapse={ policyRows.onCollapse }
-                                onSelect={ policyRows.onSelect }
-                                actionResolver={ tableActionsResolver }
-                                loading={ isLoading }
-                                error={ policyTableErrorValue }
-                                onSort={ sort.onSort }
-                                sortBy={ sort.sortBy }
-                                linkToDetailPolicy={ true }
-                            />
-                        </PolicyToolbar>
-                    </Section>
-                )}
+
+                ) : getPoliciesQuery.hasPolicies === false && hasSystems  ?
```

Comment:
I think this `hasSystems` is redundant - it wasn't `false` in previous condition.

```suggestion
                ) : getPoliciesQuery.hasPolicies === false ?
```

Reply:
Fair enough